nx.json

```diff
@@ -0,0 +1,86 @@
+{
+  "tasksRunnerOptions": {
+    "default": {
+      "runner": "@nrwl/nx-cloud",
+      "options": {
+        "cacheableOperations": [
+          "build",
+          "lint",
+          "typecheck",
+          "test-unit",
+          "test-build",
+          "test-docs",
+          "docs-build"
+        ],
+        "accessToken": "NjYwZGFlNWYtZGYyNS00MTIzLTkzYTEtYzAxZTZiYzUwYTVifHJlYWQtd3JpdGU="
```

Comment:
as mentioned in my general comment, this key must not provide write access to the cache or it would be far too dangerous to use.